content/applications/websites/ecommerce/b2b_b2c.rst

```diff
@@ -0,0 +1,133 @@
+only their products instead. To configure a B2B-specific ecommerce shop:
+
+#. Go to :menuselection:`Website --> Configuration --> Settings`, scroll down to :guilabel:`Shop
+   - Products`, and under :guilabel:`Display Product Prices` select :guilabel:`Tax-Excluded`.
+#. In that same section, enable the :ref:`Prevent Sale of Zero Priced Product
+   <ecommerce/prices/hide-prices>` feature.
+
+   .. note::
+      The redirect link and the corresponding button can be :ref:`customized
+      <ecommerce/checkout/prevent-sale>`.
+
+#. Configure a :ref:`pricelist <ecommerce/prices/pricelists>` with all prices set to `0.00`.
+#. Ensure the pricelist is assigned to the correct website and is listed first among the website's
+   pricelists.
+
+.. important::
+   To make sure that only customers with an :ref:`account and granted access
+   <ecommerce/customer_accounts/account-creation>` can see the prices, go to
+   :menuselection:`Website --> eCommerce --> Customers`, click the :icon:`fa-caret-down`
+   :guilabel:`(dropdown)` icon from the search bar, and select the :guilabel:`Archived` filter.
+   Then click the :guilabel:`Public user` card, go to the :guilabel:`Sales & Purchase` tab, and
+   add the zero-priced pricelist configured to :ref:`cover all countries
+   <ecommerce/prices/country-groups>`. Keep the :guilabel:`Public User` contact :guilabel:`Archived`
+   at all times. Website visitors now see the products without prices, while only customers with an
+   :ref:`account invitation <ecommerce/customer_accounts/account-creation>` and an assigned
+   :ref:`pricelist <ecommerce/prices/pricelists>` can view the pricing in the customer portal.
+
+.. seealso::
+   - :doc:`/applications/finance/accounting/taxes/B2B_B2C`
+   - :ref:`Discounts <ecommerce/prices/discounts>`
+
+Access request
+==============
+
+When running a B2B business, you usually :ref:`hide the pricing <ecommerce/b2b_b2c/prices>` on the
+web shop and make it available for :ref:`logged-in users <ecommerce/customer_accounts/shop-access>`
+only. To prevent anyone from signing up freely, set the :ref:`Customer Account
+<ecommerce/customer_accounts/account-creation>` setting option to :guilabel:`On inviation`.
+To create a page where customers request access, open the :doc:`website editor
+<../website/web_design>`, :ref:`create a form <website/building_blocks/form>`, customize it, and in
+the :guilabel:`Action` field, select :guilabel:`Create a customer`.
+
+.. tip::
+   Use tags to quickly identify customers who have submitted the form. To do so, add
+   a :icon:`fa-plus` :guilabel:`Field` to the form, and under :guilabel:`Type`, select
+   :guilabel:`Tags`. Remove any other unnecessary tags by clicking :icon:`fa-minus` under the
+   :guilabel:`Checkbox List`. Toggle only the tags related to B2B/B2C account requests to
+   automatically check the buttons and assign the tags when a customer fills in the form. Set the
+   :guilabel:`Visibility` to :guilabel:`hidden`.
```

Comment:
```suggestion
   :guilabel:`Checkbox List`. Toggle only the tags related to B2B/B2C account requests to
   automatically check the buttons and assign the tags when a customer fills in the form. Set the field's
   :guilabel:`Visibility` to :guilabel:`hidden`.
```
I think that if you only toggle on the tag(s) you need and set the field's visibility to hidden, you don't need to remove unnecessary tags.